padertorch/contrib/examples/audio_synthesis/wavenet/train.py

```diff
@@ -0,0 +1,94 @@
+"""
+Example call:
+
+export STORAGE_ROOT=<your desired storage root>
+python -m padertorch.contrib.examples.wavenet.train
+"""
+from pathlib import Path
+
+from padercontrib.database.librispeech import LibriSpeech
+from padertorch.contrib.examples.audio_synthesis.wavenet.data import prepare_dataset
+from padertorch.contrib.examples.audio_synthesis.wavenet.model import WaveNet
+from padertorch.io import get_new_storage_dir
+from padertorch.train.optimizer import Adam
+from padertorch.train.trainer import Trainer
+from sacred import Experiment, commands
+
+ex = Experiment('wavenet')
+
+
+@ex.config
+def config():
+    training_sets = ['train_clean_100', 'train_clean_360', 'train_other_500']
+    validation_sets = ['dev_clean', 'dev_other']
```

Comment:
In the README, you wrote that it was trained on train_clean_100 + train_clean_360. Is it reasonable to also train on noisy data? The most use cases I know of are concerned with clean speech.

Reply:
In the README I show results when training only with clean (which however are from an old model and may be updated at some point). I will add results when training on all data as soon as I have them. Do you think it is disadvantageous to add noisy data in the training?

Reply:
Not necessarily disadvantageous, but just rarely seen in cases where WaveNets are applied like TTS and VC (which use even cleaner data than LibriSpeech clean for training). If you report the results of both experiments (with and without train_other_500), I have no complaints about this.